data/json/monsters/zed_misc.json

```
@@ -1,4 +1,85 @@
 [
+  {
+    "id": "mon_frog_mother",
+    "type": "MONSTER",
+    "name": { "str": "frog mother" },
+    "description": "A large, bulbous creature with an extremely large mouth and small, stubby legs with no feet.  It is covered in an oily greenish-black residue and large, white pustules that seem ready to pop.",
+    "default_faction": "zombie",
+    "species": [ "ZOMBIE", "AMPHIBIAN" ],
+    "volume": "240 L",
+    "weight": "300 kg",
+    "hp": 600,
+    "speed": 75,
+    "material": [ "flesh" ],
+    "symbol": "F",
+    "color": "black_green",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 1,
+    "melee_dice": 1,
+    "melee_dice_sides": 4,
+    "melee_bash": 5,
+    "dodge": 2,
+    "armor_bash": 12,
+    "armor_cut": 6,
+    "armor_bullet": 1,
+    "vision_night": 7,
+    "harvest": "zombie",
+    "special_attacks": [
+      {
+        "type": "spell",
+        "spell_data": { "id": "frog_mother_summon" },
+        "cooldown": 20,
+        "monster_message": "Some of the pustules pop, and the frog mother opens her mouth and vomits tadpoles!"
+      }
+    ],
+    "death_function": [ "NORMAL" ],
+    "regenerates": 40,
+    "regen_morale": true,
+    "flags": [
+      "SEES",
+      "HEARS",
+      "SMELLS",
+      "KEENNOSE",
+      "STUMBLES",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "POISON",
+      "GRABS",
+      "NO_BREATHE",
+      "REVIVES",
+      "PUSH_MON",
+      "FILTHY"
+    ]
+  },
+  {
+    "id": "mon_tadpole_grabber",
+    "type": "MONSTER",
+    "name": { "str": "tadpole constrictor" },
+    "description": "The only resemblence to a tadpole is its slick black-green skin and its slender body.  Its three appendages, resembling long tentacles or pseudopods, propel it across the ground at a maddening speed.  It has no outward sensory organs or a mouth, and steam rises from its skin as if it is burning away its own lifeforce to catch you.",
+    "default_faction": "zombie",
+    "species": [ "ZOMBIE", "AMPHIBIAN" ],
+    "volume": "10 L",
+    "weight": "18 kg",
+    "hp": 20,
+    "speed": 200,
+    "material": [ "flesh" ],
+    "symbol": "S",
+    "color": "black_green",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 10,
+    "melee_dice": 1,
+    "melee_dice_sides": 2,
+    "dodge": 4,
+    "vision_day": 7,
+    "vision_night": 7,
+    "harvest": "exempt",
+    "special_attacks": [ [ "GRAB", 1 ] ],
+    "death_function": [ "MELT" ],
```

```suggestion
    "death_function": [ "ACID", "MELT" ],
```
I dunno. How much do you hate players? I like the idea of these things melting into a small puddle of corrosive goo, myself.